.github/workflows/release-new-version.yml

```diff
@@ -0,0 +1,122 @@
+name: Release new version
+run-name: New release for "${{ github.ref_name }}" triggered by ${{ github.actor }}
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+      tag_suffix:
```

Comment:
Maybe drop the `tag_suffix` and just use `environment` instead? Image tags like `branchname-staging-fidl` and `branchname-production-fidl` are fine, we don't need shortcuts and it will make it simpler and less likely to mess up here in CI.